README.md

```diff
@@ -1,36 +1,65 @@
+
 We happy to consider any contributions/feature requests that you may have. Please submit a PR with your changes and we will take a look. You can also use the `Github Edit` buttons available on all the website pages to help locate the file you wish to edit/raise and issue about.
-*****
-## Contents
-- [Overview](#overview)
-- [Adding a Content](#adding-a-content)
-- [Adding Redirects](#adding-redirects)
-- [Building the static site locally](#building-locally)
-- [Contributing Guide](#contributing)
-*****
-# Overview
-This website was developed by Linaro and utilises [Jekyll](https://jekyllrb.com), which is a static website generator, to provide a quick and responsive website. This website is part of a CI (Continuous Integration) build which happens on [bamboo.linaro.org](https://bamboo.linaro.org); so if you'd like to see the website builds take place take a look.
+
+## Table of Contents
+
+<!-- TOC -->
+
+- [mlplatform.org Static Jekyll Website](#mlplatformorg-static-jekyll-website)
+  - [Contributions](#contributions)
+  - [Table of Contents](#table-of-contents)
+  - [Website Theme](#website-theme)
+  - [Useful basics](#useful-basics)
+  - [Adding a page](#adding-a-page)
+    - [Step 1 - Fork the repo](#step-1---fork-the-repo)
+    - [Step 2 - Create a file](#step-2---create-a-file)
+    - [Step 3 - Add Jekyll front matter to your new page](#step-3---add-jekyll-front-matter-to-your-new-page)
+      - [Available front matter options](#available-front-matter-options)
+      - [Example front matter](#example-front-matter)
+  - [Adding a blog post](#adding-a-blog-post)
+    - [Step 1 - Modify the post file name](#step-1---modify-the-post-file-name)
+    - [Step 2 - Modify the post front matter](#step-2---modify-the-post-front-matter)
+      - [Author](#author)
+      - [Date](#date)
+      - [Image](#image)
+      - [Tags](#tags)
+      - [Description](#description)
+    - [Step 3 - Add your post content](#step-3---add-your-post-content)
+      - [Adding images](#adding-images)
+      - [Adding code](#adding-code)
+      - [Adding Media/YouTube videos](#adding-mediayoutube-videos)
+  - [Adding Redirects to the Static site](#adding-redirects-to-the-static-site)
+  - [Building the static site](#building-the-static-site)
+    - [Build instructions](#build-instructions)
+  - [Issues](#issues)
+  - [Known Issues](#known-issues)
+    - [Image file names](#image-file-names)
+
+<!-- /TOC -->
 
 ## Website Theme
+
 This website uses the `jumbo-jekyll-theme` developed as an open source project by Linaro to ensure content-driven websites are as lightweight as possible. Head over to the [theme repo] to find out more. Here are a few examples of websites using the theme:
-- [96Boards.org](https://www.linaro.org)
-- [Linaro.org](https://www.96boards.org)
-- [connect.linaro.org](https://connect.linaro.org)
-- [OP-TEE.org](https://www.op-tee.org)
-- [TrustedFirmware.org](https://www.trustedfirmware.org)
-- [DeviceTree.org](https://www.devicetree.org)
+
```

Comment:
This is an example of why I think this should be centralised and just referenced from the README, rather than replicating this list on every README on every website. Too little consistency on how we tell contributors what to do.